### PR TITLE
Cleanup remaining runlevel references

### PR DESCRIPTION
**Pull request by M4rtinK**
We have been using systemd targets for quite some time already,
so cleanup the remaining runlevel references from the code.

### CODE REVIEW

File: pyanaconda/desktop.py [hunk 1]
```diff
@@ -18,39 +18,42 @@
...
 
-    @runlevel.setter
-    def runlevel(self, runlevel):
-        if int(runlevel) not in RUNLEVELS:
-            raise RuntimeError("Desktop::setDefaultRunLevel() - Must specify runlevel as one of %s" % RUNLEVELS.keys())
+    @default_target.setter
+    def default_target(self, target):
+        supported_targets = [TEXT_ONLY_TARGET, GRAPHICAL_TARGET]
+        if target not in supported_targets:
+            raise RuntimeError("Desktop::default_target - Must specify a systemd default target as one of %s" % supported_targets)
```
Comment (jkonecny12) [inline pyanaconda/desktop.py]: Please split this row.

**Comment by jkonecny12** [comment]
Other than that the small cosmetic change above, it looks good to me.

File: pyanaconda/desktop.py (hunk 1 above)
Comment (M4rtinK) [inline pyanaconda/desktop.py]: Done. :)

**Comment by M4rtinK** [comment]
Thanks for review, pushed! :)

**Pull request closed without merging**